.travis.yml

```diff
@@ -0,0 +1,19 @@
+notifications:
+  email: false
+
+sudo: true
+
+language: python
+
+python:
+  - "3.6"
+
+services:
+  - docker
+
+before_script:
+  - gem install awesome_bot
+  - pip install mkdocs mkdocs-material
```

Comment:
Can you introduce `requirements.txt` so that we can centralise everything there and include it in dockerfile, travis ci, etc?
(as in www.reana.io repository)

Reply:
Can you also enrich README.md with a few About words and with example how to run the site locally?  Something like:

```console
$ mkvirtualenv docs.reana.io
$ pip install -r requirements.txt
$ mkdocs build
$ firefox site/index.html
```